### PR TITLE
Add mini

### PR DESCRIPTION
**Pull request by LezaiYa**
It's been tested for a long time and it's more stable

### CODE REVIEW

**Comment by Loonie-Toons** [comment]
You couldn't have made this an add-on so you can be a Mini Sheriff or whatever?

**Comment by NikoCat233** [comment]
> You couldn't have made this an add-on so you can be a Mini Sheriff or whatever?

It orginally a main role in tohexi
You mean addon mini that cannot be killed before age 18?

**Comment by Loonie-Toons** [comment]
> > You couldn't have made this an add-on so you can be a Mini Sheriff or whatever?
> 
> It orginally a main role in tohexi
> You mean addon mini that cannot be killed before age 18?

It's better as an add-on as being a basic ass crewmate that makes everyone lose when you die is boring and lame.

As for the Mini Impostor, it's a plain ass Impostor.

**Comment by NikoCat233** [comment]
The main concept is that crew got to find out whether mini is evil or not
They got to vote out evil mini, but they lose for voting out nice mini
Also nice mini can help find out imp as they are invincible before age 18

**Comment by NikoCat233** [comment]
Absolutely we can make this a addon
Just make people invincible before age 18

File: Modules/NameColorManager.cs [hunk 1]
```diff
@@ -177,8 +181,10 @@ private static bool KnowTargetRoleColor(PlayerControl seer, PlayerControl target
             || (target.Is(CustomRoles.Mayor) && Options.MayorRevealWhenDoneTasks.GetBool() && target.GetPlayerTaskState().IsTaskFinished)
             || (seer.Is(CustomRoleTypes.Crewmate) && target.Is(CustomRoles.Marshall) && target.GetPlayerTaskState().IsTaskFinished)
          //   || Mare.KnowTargetRoleColor(target, isMeeting)
-            || EvilDiviner.IsShowTargetRole(seer, target)
-            || PotionMaster.IsShowTargetRole(seer, target);
+            || (target.Is(CustomRoles.NiceMini) && Mini.EveryoneCanKnowMini.GetBool())
```
Comment (Tommy-XL) [inline Modules/NameColorManager.cs:184]: Why **|| EvilDiviner.IsShowTargetRole(seer, target) || PotionMaster.IsShowTargetRole(seer, target)** was deleted, lol

Comment (Loonie-Toons) [reply Modules/NameColorManager.cs:184]: On it

Comment (Loonie-Toons) [reply Modules/NameColorManager.cs:184]: Fixed

**Review by NikoCat233 — COMMENTED** [review]
Need to sync minicrew age from host for mod players
or age will always be 0 for them
https://github.com/Loonie-Toons/TownOfHost-ReEdited/pull/427/commits/e69c855d421673f1e6f112144c207211646a3c31

**Comment by NikoCat233** [comment]
> Need to sync minicrew age from host for mod players or age will always be 0 for them [e69c855](https://github.com/Loonie-Toons/TownOfHost-ReEdited/commit/e69c855d421673f1e6f112144c207211646a3c31)

Mci does not sync like real client

**Comment by LezaiYa** [comment]
> You couldn't have made this an add-on so you can be a Mini Sheriff or whatever?

If there are any problems, please ask Loonie to help me fix them because I am currently at school and I cannot play with the computer

**Pull request merged**